libbeat/docs/config-file-format.asciidoc

```diff
@@ -370,7 +333,15 @@ use this, for example, for setting defaults in a base configuration file, and
 overwrite settings via local configuration files.
 
 In addition to overwriting settings using multiple configuration files,
-individual settings can be overwritten using `-E <setting>=<value>`.
+individual settings can be overwritten using `-E <setting>=<value>`. The
+`<value>` can be either a single value or a complex object, such as a list or
+dictionary. For example, you can disable the Elasticsearch output and write all
+events to the console by setting:
+
+["source","sh"]
+--------------------------------------------------------------------------------
+-E output='{elasticsearch.enabled: false, console.pretty: true}'
+--------------------------------------------------------------------------------
```

Comment:
maybe include sample config and final config here?

e.g. with `-E output='{elasticsearch.enabled: false, console.pretty: true}'`

and a config file like:

```
output.elasticsearch:
  hosts: ["http://localhost:9200"]
  username: username
  password: password
```

then the final configuration presented to the beat becomes:

```
output.elasticsearch:
  enabled: false
  hosts: ["http://localhost:9200"]
  username: username
  password: password

output.console:
  pretty: true
```

That is, overwriting fields with complex objects will merge the object with the original configuration, not changing/overwriting any other settings.